packages/react-composites/src/composites/index.ts

```diff
@@ -3,5 +3,7 @@
 
 export * from './ChatComposite';
 export * from './CallComposite';
-export * from './OneToOneCall';
 export { ErrorBar } from './common';
+
+// Export for storybook, these being exported should be re-evaluated when composites are pure under the new architecture.
+export * from './IncomingCallAlerts';
```

Comment:
Are these used anywhere in the composite dir? Should they just be moved to the examples in storybook and out of this dir?

Reply:
Sure can do as follow up.

Didn't look deep enough to know if there are dependencies we'll have to untangle.